saleor/core/utils/__init__.py

```diff
@@ -46,7 +46,7 @@ def build_absolute_uri(location, is_secure=False, site_settings=None):
     # type: (str, bool, saleor.site.models.SiteSettings) -> str
     if site_settings is None:
         site_settings = get_site_settings()
-    host = site_settings.domain
+    host = site_settings.site.domain
```

Comment:
You can get Site directly from contrib.sites

Reply:
Done